### PR TITLE
PDC-947,the boolean values for filters should not be case sensitive

### PR DESCRIPTION
**Pull request by bliuredhat**
Make bootable filter on images case-insensitive

JIRA: PDC-947

### CODE REVIEW

**Comment by lubomir** [comment]
Generally looks good. I would like the filter to actually be called something like `CaseInsensitiveBooleanFilter`, but that is likely out of scope for this issue. One minor thing: in order to have nice git history, please change the commit message a bit – put the issue number in the body. I would also suggest different wording:
```
Make bootable filter on images case-insensitive

JIRA: PDC-947
```
These two links have some guidelines about good commit messages. These are not hard rules, just something to think about:
http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
http://chris.beams.io/posts/git-commit/

**Comment by bliuredhat** [comment]
lubomir, thank you so much~! The link you surported is very good for me.  I will rewrite the commit message.

**Comment by bliuredhat** [comment]
lubomir, The 'CustomizeBooleanFilter' function in [pdc/apps/package/filters.py] is not defined by me, so I can't redefine the name.

**Comment by erichuanggit** [comment]
Looks good to me.

File: requirements/devel.txt [hunk 1]
```diff
@@ -10,6 +10,6 @@ setuptools
 lxml
 # model graph
 # requires: graphviz-devel
-pygraphviz
+pygraphviz<1.3
```
Comment (erichuanggit) [inline requirements/devel.txt]: Remove this change, since it has been fixed in pygraphviz 1.3.1

**Pull request merged**